packages/jds/src/utils/PSEUDO_ELEMENT_POLICY.md

```
@@ -1,204 +1,136 @@
-// 케이스 2: 시각 영역 < 탭 영역 (IconButton condensed 등)
-selectors: {
-  "&::before, &::after": { inset: pxToRem(-4), borderRadius: "4px" },
-}
+- hover는 root 자신의 `:hover`로 판단한다. 커서가 컴포넌트 위에 있으면 보조 액션 위에 있을 때도 root overlay가 켜지고, 보조 액션은 그 위에 자기 overlay를 겹쳐 그린다.
+- press와 focus는 내부에 `data-interaction-target`을 부여한 버튼의 상태만 읽는다. 보조 액션을 누르거나 포커스해도 root는 반응하지 않는다.
+
+hover는 커서의 위치를 나타내고 press와 focus는 어떤 컨트롤을 조작하는지를 나타낸다. 그래서 hover는 표면 기준으로, press와 focus는 컨트롤 기준으로 잡는다. 이것은 File에만 적용하는 예외가 아니라 이런 구조 전체에 적용하는 규칙이다.
+
+`interaction: "delegated"`가 이 규칙을 구현한다. overlay는 hover를 자기 요소로 두고 press만 내부 버튼으로 넘기며, focusRing은 focus를 내부 버튼으로 넘긴다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)